main_test.go

```diff
@@ -1,23 +1,27 @@
-package dothill
+package exosx
 
 import (
 	"fmt"
 	"os"
 	"testing"
 
+	"github.com/joho/godotenv"
 	. "github.com/onsi/gomega"
 )
 
 var client = NewClient()
 
 func init() {
-	client.Addr = "http://localhost:8080"
-	client.Username = "manage"
-	client.Password = "!manage"
-
-	if endpoint := os.Getenv("API_ENDPOINT"); endpoint != "" {
-		client.Addr = endpoint
+	settingsfile := ".env"
+	err := godotenv.Load(settingsfile)
+	if err != nil {
```

Comment:
Does this behave properly if the .env file does not exist and credentials are provided via environment variables?

Reply:
No, good feedback. I updated the tests to work properly with or without an .env file. I also verified that exporting ENV variables will override any .env file setting. Also updated the tests to work properly with older and newer API versions.

Reply:
No, good point. The code was updated to work with or without and .env file. I also updated the code to always use ENV variables over any value set in the .env file. Tests passed with older and newer API versions.